npm/private/pnpm.bzl

```diff
@@ -485,8 +508,20 @@ def _convert_v9_packages(packages, snapshots):
         # package_data can have the resolved "version" for things like https:// deps
         friendly_version = package_data["version"] if "version" in package_data else static_key[version_index + 1:]
 
+        package_id = package_snapshot.get("id", None)
```

Comment:
Do we have any tests that show the side-effects of this change? Can we see that in a diff?